torch/csrc/jit/python/init.cpp

```diff
@@ -360,7 +361,7 @@ void initJITBindings(PyObject* module) {
             checkAliasAnnotation(g, std::move(stack), unqualified_op_name);
           })
       .def(
-          "_jit_register_cuda_fuser", &registerCudaFuseGraph)
+          "_jit_register_cuda_fuser", &RegisterCudaFuseGraph::registerPass)
```

Comment:
You might want to expose `RegisterCudaFuseGraph::clearPass` as well.

Reply:
I need to fix merge conflicts in the big fuser PR. I'll do this then, so I can also test it. Let me know if you prefer it here.